.travis.yml

```diff
@@ -10,7 +10,10 @@ matrix:
 before_install:
     - pip install pipenv
     - pipenv install --dev --skip-lock
+    - pip install codespell flake8
```

Comment:
Could you move these dependencies into the `Pipfile` under the dev dependencies.

Reply:
Don't forget to remove this please 🙂